uvicorn/protocols/http/h11_impl.py

```diff
@@ -360,8 +364,9 @@ def __init__(
         self.transport = transport
         self.flow = flow
         self.logger = logger
-        self.access_logger = logger
```

Comment:
this is not part of the PR but it triggered a bug in my new test so i figured i'd fix it